packages/palette-docs/plugins/gatsby-plugin-local-algolia-docsearch/gatsby-ssr.js

```diff
@@ -28,3 +28,17 @@ exports.onRenderBody = (
     />,
   ])
 }
+
+exports.onPreRenderHTML = ({ getHeadComponents, replaceHeadComponents }) => {
+  const headComponents = getHeadComponents()
+  // Move `docsearch-css` to the top so that our custom styles override it.
+  headComponents.sort((x, y) => {
```

Comment:
#superminor - x / y generally refers to positioning coordinates, where a / b refers to sequence, or more general things.